github/tests/ReplayData/Github.testFindOrganizationInstallation.txt

```diff
@@ -0,0 +1,10 @@
+https
+GET
+api.github.com
+None
+/orgs/arda/installation
+{'Authorization': 'Basic login_and_password_removed', 'User-Agent': 'PyGithub/Python', 'Accept': 'application/vnd.github.machine-man-preview+json'}
```

Comment:
This API does not use `Basic` auth, test works because this is mock data. There's no simple way to toggle `self.g` in `Framework.py`, so maybe this is fine.